aiohttp/helpers.py

```diff
@@ -876,6 +878,14 @@ def set_cookie(
         if samesite is not None:
             c["samesite"] = samesite
 
+        if DEBUG:
+            cookie_length = len(c.output(header="")[1:])
```

Comment:
What exactly are you cutting off with `[1:]`? A space? Then, it'd be more readable to use `str.lstrip()` to trim it.

Reply:
If not, then at least add a code comment explaining this magic.

Reply:
I think `lstrip()` could potentially cause issues if a cookie started with whitespace or something. I suspect the output code produces something like "{header} {cookie}", so there's an extra dividing character that doesn't count towards the cookie.

Could also just count 1 less: `len(c.output(header="")) - 1`.

Reply:
I agree with @Dreamsorcerer, if the cookie has more than one extra space character `lstrip()` will remove them which is not what we want here.